03-deploying_shiny_app.Rmd

```diff
@@ -182,7 +182,7 @@ First `cd` into the directory containing the Dockerfile.  Then run:
 
 `docker build . -t your_name_here`    (`your_name_here` is the tag you want to give to the image)
 
-By building the image, it will test the installation of of the packrat dependencies into an identical Linux environment to what's on the platform.
+By building the image, it will test the installation of the packrat dependencies and package your shiny app into a Linux environment identical to what's on the platform.
```

Comment:
To be honest here I was tempted to say "almost identical" rather than "identical" as by the nature of re-building a docker image things may be slightly differ.

e.g. base image may change, or some version of the system packages installed may be slightly different at the time of build locally vs. on the platform.

Basically, once you build a Docker image it's set in stone but re-building an image from the exact same `Dockerfile` may result in slightly different results.

But I realise this is a bit pedantic and the final user probably don't care much about this technicalities. Sorry 💃